### ISSUE TITLE
Auto Login

### ISSUE THREAD

**Issue by Shiemy** [xan105/Aion-Launcher#2]
Hey, just a short Question, you said:
*2 Equivalent of using command line parameter -user -password. Launcher will keep your credentials crypted (aes-256-cbc) on your computer but they will be passed clearly to aion.bin so use this at your own risk.

so if i used it it will passed to the Aion bin , so how can i remove it from the bin? also WHICH bin file ^^'
i Use Steam Aion and simply tested this feature, but i am also Aware to my data :D 

so in short:
is there a way to delete my "Login Info" if yes how ^^

**Comment by xan105** [comment]
The bin refers to the game executable "aion.bin" which has an auto login feature if you run it with 
```-user xxx -password yyy``` arguments.

In my launcher there is a login form :

1
- If it is empty then it will launch the game as usual.
- If it is not empty then it will launch Aion with ```-user xxx -password yyy``` provided in said form.

2
- If you check the checkbox "Remember login" it will save user and password in ```%localappdata%\AION-LIVE\``` and it will crypt it in AES so that it is not written in clear.
What I mean by *"but they will be passed clearly to aion.bin"* is even tho it is crypted in the file if you go look for the command line passed to the game executable "aion.bin" in the task manager you will see your credentials in clear text.
- If you uncheck the checkbox it will erase any credentials saved in ```%localappdata%\AION-LIVE\``` whether the form is empty or not.

And if the launcher doesn't find any credentials then the form will be empty on startup.
Changes are applied after you press the Play button.

**So in short** : make sure you run the game at least once with the checkbox unchecked and that the form is empty on startup when you start the launcher the next time.

Note that you could also delete the entire  ```%localappdata%\AION-LIVE\``` folder to achieve this 😅 but it is rather brutal and will probably break something within my launcher.

I hope this was clear enough.
